src/app/(pages)/groups/components/table/options/remove-membership-button/index.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+import { useState } from "react";
+import ConfirmUnlinkUserModal from "./modal";
+import { ScimReference } from "@/models/scim";
+
+export type RemoveMembershipProps = {
+  userRef: ScimReference;
+  groupId: string;
+  groupName: string;
+};
+
+export default function RemoveMembership(
+  props: Readonly<RemoveMembershipProps>
+) {
+  const { userRef, groupId, groupName } = props;
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const showModal = () => setIsModalOpen(true);
+  const closeModal = () => setIsModalOpen(false);
```

Comment:
Not a big issue, but to be super consistent I think that in other modals I simply use

```ts
  const [show, setShow] = useState(false);
  const open = () => setShow(true);
  const close = () => setShow(false);
  ```
  See https://github.com/indigo-iam/iam-dashboard/blob/c146b7ba9925688db64421d2cef79ba9c6dd82a7/src/app/(pages)/requests/components/groups/options/confirm-button.tsx#L13